### PR TITLE
E2e/feature/ri 5138 dry run job

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    })
+    .afterEach(async() => {
+        // delete instances via UI
+        await rdiInstancesListPage.deleteAllInstance();
+    });
+test('Verify that user can use Dry run panel', async() => {
+    const instanceId = 'testId';
+    const instanceUrl = 'http://localhost:4000';
+    // const job = 'testJob';
```
Comment (mariasergeenko) [inline tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts:38]: should be commented?

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    const job = 'job2';
+
+    await rdiInstancesListPage.addRdi(rdiInstance);
```
Comment (mariasergeenko) [inline tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts:41]: you can do it via API

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    .meta({ type: 'critical_path' })
+    // it will be removed
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await t.maximizeWindow();
+
+    })
+    .afterEach(async() => {
+        // delete instances via UI
+        await rdiInstancesListPage.deleteAllInstance();
```
Comment (mariasergeenko) [inline tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts:33]: can remove it via Api

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts (hunk 2 above)
Comment (mariasergeenko) [reply tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts:41]: as I remember, I've already implemented it

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    });
+test('Verify that user can use Dry run panel', async() => {
+    const instanceId = 'testId';
+    const instanceUrl = 'http://localhost:4000';
+    // const job = 'testJob';
+    const job = 'job2';
+
+    await rdiInstancesListPage.addRdi(rdiInstance);
+    await updateColumnValueInDBTable(rdiTable, 'id', instanceId);
+    await updateColumnValueInDBTable(rdiTable, 'url', instanceUrl);
```
Comment (mariasergeenko) [inline tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts:43]: the same as comment from another request, how we can specify exact row to update?

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts]: updated, deleted

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts (hunk 2 above)
Comment (vlad-dargel) [reply tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts]: yes, thanks, switched to api methods

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts (hunk 3 above)
Comment (vlad-dargel) [reply tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts]: updated

File: tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts (hunk 4 above)
Comment (vlad-dargel) [reply tests/e2e/tests/web/critical-path/rdi/dry-run-job.e2e.ts]: here just 1st row updated
will be changed in the other pr

**Review by mariasergeenko — APPROVED** [review]

**Pull request merged**